### ISSUE TITLE
Default window does not show home switch state on some computers.

### ISSUE THREAD

**Issue by rlwoodjr** [winder/Universal-G-Code-Sender#652]
Several of my customers are not able to get all of the "Controller State (DRO)" window to show upi on the screen with the default windows setting. This hides the XYZ home state, probe, door, hold and other check boxes,. I would assume that their screen resolution is not high enough

**Comment by winder** [comment]
@rlwoodjr yes this is a screen resolution (and UX) problem. For now they can drag the windows around if needed. I would like to redesign this panel again to make the data fit better, I have the same problem on my computer.

**Comment by breiler** [comment]
It's not optimal, but I've at least added scroll bars to the DRO window in #1050
